test-data/unit/check-modules.test

```diff
@@ -323,11 +323,10 @@ main:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missin
 main:2: error: Name 'y' is not defined
 
 [case testUnknownModuleRedefinition]
-import xab
+# Error messages differ with the new analyzer
+# flags: --no-new-semantic-analyzer
```

Comment:
We generally have preferred to use `--new-semantic-analyzer` if test case passes with both semantic analyzers, but error messages differ.